<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>usuarios</class>
 <widget class="QDialog" name="usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión usuarios</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTableView" name="usuariostableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="nuevopushButton">
       <property name="text">
        <string>&amp;Nuevo</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/Files-New-File-icon.png</normaloff>:/iconos/graf/Files-New-File-icon.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editarpushButton">
       <property name="text">
        <string>&amp;Editar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/lapiz.xpm</normaloff>:/iconos/graf/lapiz.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eliminarpushButton">
       <property name="text">
        <string>E&amp;liminar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButto">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>usuariostableView</tabstop>
  <tabstop>nuevopushButton</tabstop>
  <tabstop>eliminarpushButton</tabstop>
  <tabstop>salirpushButto</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButto</sender>
   <signal>clicked()</signal>
   <receiver>usuarios</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
